[0m23:08:17.546627 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022416D61D30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000224151991D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022417E87ED0>]}


============================== 23:08:17.551243 | 50ad6bd0-e835-493f-abe9-fa9e6aac4b82 ==============================
[0m23:08:17.551243 [info ] [MainThread]: Running with dbt=1.9.2
[0m23:08:17.551952 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\Roberto Leonie\\.dbt', 'version_check': 'True', 'warn_error': 'None', 'log_path': 'logs', 'fail_fast': 'False', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init dbt_project', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m23:08:17.566017 [debug] [MainThread]: Starter project path: C:\Users\Roberto Leonie\AppData\Local\Programs\Python\Python313\Lib\site-packages\dbt\include\starter_project
[0m23:08:17.574467 [info ] [MainThread]: 
Your new dbt project "dbt_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m23:08:17.575087 [info ] [MainThread]: Setting up your profile.
[0m23:12:04.061696 [info ] [MainThread]: Profile dbt_project written to C:\Users\Roberto Leonie\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m23:12:04.062718 [debug] [MainThread]: Command `dbt init` succeeded at 23:12:04.062566 after 226.71 seconds
[0m23:12:04.063162 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002241805C9D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000224180367B0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022417D46CF0>]}
[0m23:12:04.063580 [debug] [MainThread]: Flushing usage events
[0m23:12:05.057457 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m23:12:10.692940 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000138AB591D30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000138A99FD1D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000138AC6B7ED0>]}


============================== 23:12:10.696306 | 2f66b984-bf38-49cb-a32f-3af6d1dea995 ==============================
[0m23:12:10.696306 [info ] [MainThread]: Running with dbt=1.9.2
[0m23:12:10.696997 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\Roberto Leonie\\.dbt', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'warn_error': 'None', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt debug', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m23:12:10.706955 [info ] [MainThread]: dbt version: 1.9.2
[0m23:12:10.707451 [info ] [MainThread]: python version: 3.13.0
[0m23:12:10.707756 [info ] [MainThread]: python path: C:\Users\Roberto Leonie\AppData\Local\Programs\Python\Python313\python.exe
[0m23:12:10.708112 [info ] [MainThread]: os info: Windows-11-10.0.26100-SP0
[0m23:12:10.763529 [info ] [MainThread]: Using profiles dir at C:\Users\Roberto Leonie\.dbt
[0m23:12:10.764131 [info ] [MainThread]: Using profiles.yml file at C:\Users\Roberto Leonie\.dbt\profiles.yml
[0m23:12:10.764544 [info ] [MainThread]: Using dbt_project.yml file at C:\Users\Roberto Leonie\Desktop\Desafio Dados.Rio\dbt_project.yml
[0m23:12:10.764987 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m23:12:10.765792 [debug] [MainThread]: Command `dbt debug` failed at 23:12:10.765688 after 0.26 seconds
[0m23:12:10.766068 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000138AC9916E0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000138AC9B44D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000138AC8A6690>]}
[0m23:12:10.766358 [debug] [MainThread]: Flushing usage events
[0m23:12:11.322219 [debug] [MainThread]: An error was encountered while trying to flush usage events
